gcloud/datastore/batch.py

```diff
@@ -21,7 +21,7 @@ class Local(object):
 
 from gcloud.datastore import _implicit_environ
 from gcloud.datastore import helpers
-from gcloud.datastore import datastore_v1_pb2 as datastore_pb
+from gcloud.datastore import _datastore_v1_pb2 as datastore_pb
```

Comment:
Doesn't making the module "private" work against @jgeewax's notion that we tell folks who want features we don't support to "kick down" to it?